site/aboutIVIFoundation/default.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+nav_order:  5
+title: About the IVI Foundation
+has_children: true
+---
+
+# About IVI
+
+These pages will help you learn more about the IVI Foundation and IVI
+technology.
+
+- To understand why IVI was formed, see the [Charter](charter.html)
+  page.
+- See [Overview](overview.html) to get an overview of IVI technology
+  and benefits.
+- The [Architecture](architecture.html) page describes the overall IVI
+  software architecture, explaining the major software components and
+  standard instrument classes.
+- [Instrument Classes](instrument_classes.html) describes IVI's
+  instrument classes in more detail.
```

Comment:
Defining `instrument classes` in <5 words would be helpful in this list item. What about...

> [Instrument Classes](instrument_classes.html) describe the kinds of instruments (e.g. DMMs or switches) that IVI supports with standardized API interfaces.